Lib/smtplib.py

```diff
@@ -36,6 +37,8 @@
 # Better RFC 821 compliance (MAIL and RCPT, and CRLF in data)
 #     by Carey Evans <c.evans@clear.net.nz>, for picky mail servers.
 # RFC 2554 (authentication) support by Gerhard Haering <gerhard@bigfoot.de>.
+# Upper case commands for wider support as enumerated in section 2.4 of RFC 5321
+#     by Steve Jacob <steve@mymangomail.com>
```

Comment:
There are some change logs in a module's source code, but usually we don't update them nowadays. Instead, we mention them in the [news entry](https://devguide.python.org/contrib/core-team/committing/#updating-news-and-what-s-new-in-python) (where the PR author's name can also be credited).